file-ops.h

```diff
@@ -3,6 +3,11 @@
 #include <fstream>
 #include <iostream>
 
+#include "bgfx/platform.h"
+#if BX_PLATFORM_EMSCRIPTEN
+#include <emscripten.h>
+#endif
```

Comment:
update: See later comment for more context but this could go into its own file just for enscripten